CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 # =======================================================================
 #+
 
-cmake_minimum_required(VERSION 3.1.1)
+cmake_minimum_required(VERSION 2.8)
```

Comment:
Lowering the minimum required version will cause us trouble (especially with upcoming merges from refactoring_sandbox). We would prefer to avoid changing it.

Reply:
That's OK - I thought I'd give it a try - we can keep this in our build for now